src/libs/testing-library/setup.ts

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/jest-dom/vitest'; // to add vitest jsdom matcher support
+import { vitest } from 'vitest';
+import { tokens } from './utils';
+
+// MOCK STORE PROVIDER CONTEXTS
+
+vitest.mock('store/useTokensStore.ts', async (importOriginal) => {
+  const mod = await importOriginal<typeof import('store/useTokensStore.ts')>();
+  const tokensMap = new Map(
+    tokens.map((token) => [token.address.toLowerCase(), token])
+  );
+  return {
+    ...mod,
+    useTokensStore: () => {
+      return {
+        tokens,
+        importedTokens: [],
+        tokensMap,
+        isPending: false,
+        isError: false,
+        error: null,
+        setImportedTokens: () => {},
+      };
+    },
+  };
+});
+
+// MOCK CARBON SDK
+enum MarginalPriceOptions {
```

Comment:
I think you can hardcode it directly in the mock, no need to create an enum

Reply:
You're right, removed it